### PR TITLE
variance is a branch stat

### PR DESCRIPTION
**Pull request by petrelharp**
Finally remembered to PR my edits this time...

And, I remembered something important/useful/at least cool about the variance partitioning: please have a look at the first commit to see if it makes sense.

I will continue editing.

### CODE REVIEW

File: treestats_paper.tex [hunk 1]
```diff
@@ -1467,7 +1480,7 @@ \section*{Discussion}
 % This additional mode of reporting statistics is an important subject for future work.
 % moving past a few summaries in discrete groups to a genome/time/space view of relatedness
 The computational toolbox of population genetics
-is still mostly composed of statistics designed for analysis of a handful of loci
+is still mostly composed of statistics orignally designed for analysis of a handful of loci
```
Comment (jeromekelleher) [inline treestats_paper.tex]: typo here, orignally

File: treestats_paper.tex [hunk 2]
```diff
@@ -797,30 +797,43 @@ \section*{Duality of site and branch statistics}
...
+and can itself be written as a Branch statistic,
+using the same weights and the summary function $f(x)^2$
+\citep[Lemma 2 in][]{ralph2019empirical}.
+The reason for this is straightforward ---
+the Site statistic is a sum across branches of the number of mutations on the branch
+multiplied by a summary, $f(x)$.
+Given the trees, these numbers of mutations are independent and Poisson,
+and the variance of $f(x)$ multiplied by a Poisson random variable with mean $m$ is $m f(x)^2$.
+The sum is divided by the length of the region, $j-i$,
+and so the variance is divided by $(j-i)^2$.
```
Comment (jeromekelleher) [inline treestats_paper.tex]: I haven't checked the maths, but sounds right (and *very* neat and elegant) to me.

Comment (petrelharp) [reply treestats_paper.tex]: :+1: happy to have remembered this. =)

**Comment by petrelharp** [comment]
Ok, I've rejiggered some figures; maybe all good now?

**Pull request merged**